apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx

```diff
@@ -28,32 +29,53 @@ const ChatInputText = ({ onSubmit, sendDisabled }: ChatInputTextProps) => {
   };
 
   return (
-    <Paper
+    <Box
       component='form'
       onSubmit={handleSubmit}
       sx={{
-        p: '4px 4px',
         display: 'flex',
-        alignItems: 'center',
+        alignItems: 'end',
         width: 'auto',
+        padding: 2,
+        borderTop: '3px solid',
+        borderTopColor: message.length ? palette.primary.main : disabledColor,
+        backgroundColor: palette.background.paper,
       }}
     >
       <InputBase
         fullWidth
         placeholder={t('chatBot.writeNewMessagePlaceholder')}
         value={message}
         onChange={handleChange}
-        sx={{ ml: 1 }}
+        multiline
+        maxRows={8}
+        endAdornment={
+          <span style={{ color: alpha(palette.text.primary, 0.4) }}>
+            {message.length}/800
+          </span>
+        }
```

Comment:
![image](https://github.com/user-attachments/assets/6d1d627c-ebc6-4fe9-8040-aeed86393dca)
Char counter should be align to bottom of text field and text size should be 12px in mobile and 14px in dekstop view

Reply:
Done